base/bucket_test.go

```diff
@@ -279,10 +289,140 @@ func TestGetPoolName(t *testing.T) {
 
 func TestGetViewQueryTimeout(t *testing.T) {
 	fakeBucketSpec := &BucketSpec{}
-	expectedViewQueryTimeout := time.Duration(75*1000) * time.Millisecond
+	expectedViewQueryTimeout := time.Duration(75) * time.Second
 	assert.Equal(t, expectedViewQueryTimeout, fakeBucketSpec.GetViewQueryTimeout())
 	viewQueryTimeoutSecs := uint32(0)
 	fakeBucketSpec.ViewQueryTimeoutSecs = &viewQueryTimeoutSecs
-	expectedViewQueryTimeout = time.Duration(1000*60*60*24*365*10) * time.Millisecond
+	expectedViewQueryTimeout = time.Duration(60*60*24*365*10) * time.Second
 	assert.Equal(t, expectedViewQueryTimeout, fakeBucketSpec.GetViewQueryTimeout())
 }
+
+var (
+	rootKeyPath    = fmt.Sprintf("%vroot.key", os.TempDir())
```

Comment:
I think we need a path separator here, but also needs to be cross-platform:

`filepath.Join(os.TempDir(), "root.key")`

Reply:
We don't need a path separator her; `os.TempDir()` will have path a separator at the end, but let's use `filepath.Join(os.TempDir(), "root.key")` to make it platform independent.

Reply:
My understanding of `os.TempDir()` is that it returns the path to a temp directory, without the trailing slash/separator.

https://play.golang.org/p/8FXtRPXiQar